Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
Why cannot the constructor make the distinction between what is being passed to it and then act accordingly? (Sorry, I haven't found the time to carefully look at the code, but a cursory glance didn't reveal anything that would make this behavior impossible.)

> Yes. But how to tag it as private?

```ruby
private_class_method :new
```